src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -392,15 +392,21 @@ private FeedbackSessionAttributes cleanSessionData(FeedbackSessionAttributes ses
     * Therefore the question number corresponding to the created response 
     * should be inserted in the json file in place of the actual response ID.<br />
     * This method will then generate the correct ID and replace the field.
+     * @throws EntityDoesNotExistException 
     **/
-    private FeedbackResponseAttributes injectRealIds(FeedbackResponseAttributes response) {
+    private FeedbackResponseAttributes injectRealIds(FeedbackResponseAttributes response) throws EntityDoesNotExistException {
         try {
             int qnNumber = Integer.parseInt(response.feedbackQuestionId);
         
+            FeedbackQuestionAttributes question = feedbackQuestionsLogic.getFeedbackQuestion(
+                    response.feedbackSessionName, response.courseId,
+                    qnNumber);
+            if (question == null) {
+                throw new EntityDoesNotExistException("question has not persisted yet");
+            }
             response.feedbackQuestionId = 
-                feedbackQuestionsLogic.getFeedbackQuestion(
-                        response.feedbackSessionName, response.courseId,
-                        qnNumber).getId();
+                question.getId();
+            
```

Comment:
Judging from history, this is where the NPE that <code>AdminInstructorAccountAddAction</code> was catching came from. It's not required by PMD, but the behaviour in <code>AdminInstructorAccountAddAction</code> would change if I didn't modify it to throw a EntityDoesNotExistException if its null